scripts/utils.sh

```diff
@@ -1,21 +1,23 @@
 ################################# setup colors
 # setting up colors
-BLU='\033[0;34m'
+BOLD='\033[10;1m'
```

Comment:
Any reason bold vs blu? just to see it better?

Reply:
yes, just to see it better and differentiate from logs of other steps.